front/src/modules/scenario/components/ScenarioCard.tsx

```diff
@@ -30,8 +30,8 @@ export default function StudyCard({ setFilterChips, scenario }: Props) {
   };
 
   return (
-    <div className="scenarios-list-card">
-      <div className="scenarios-list-card-name" data-testid={scenario.name}>
+    <div className="scenarios-card">
```

Comment:
Can you rename the class scenario-card ? (and in all the sub-classes ?)
```suggestion
    <div className="scenario-card">
```